src/libs/hooks/useScrollLock/helpers/handleOverflow.js

```diff
@@ -0,0 +1,43 @@
+let previousPaddingRight;
+let previousOverflowSetting;
+
+function hiddenOverflow() {
+    setTimeout(() => {
+        if (previousPaddingRight === undefined) {
+            const scrollBarGap = window.innerWidth - document.documentElement.clientWidth;
+
+            if (scrollBarGap > 0) {
+                previousPaddingRight = document.body.style.paddingRight;
+                document.body.style.paddingRight = `${scrollBarGap}px`;
+            }
+        }
+
+        if (previousOverflowSetting === undefined) {
+            previousOverflowSetting = document.body.style.overflow;
+            document.body.style.overflow = 'hidden';
```

Comment:
I have doubts about this implementation since we have implemented a more complex here src/libs/scrollController/index.js in order to support all browser and mobile take a look at this old issue:
https://github.com/nexxtway/react-rainbow/issues/1031